src/server/symbols.map

```diff
@@ -169,6 +169,7 @@ MIR_SERVER_1.6.0 {
     mir::scene::SurfaceObserver::operator*;
     mir::scene::SurfaceObserver::?SurfaceObserver*;
     mir::scene::SurfaceObserver::SurfaceObserver*;
+    mir::scene::SurfaceStateStack::*;
```

Comment:
We've going to be releasing this with 2.4, so 1.6 is the wrong stanza for this to be added to.

Reply:
Done (I think?)